examples/provider/config-in-cluster.yaml

```diff
@@ -1,12 +1,33 @@
-# Make sure provider-kubernetes has enough permissions to install your objects into cluster
-#
-# You can give admin permissions by running:
-# SA=$(kubectl -n crossplane-system get sa -o name | grep provider-kubernetes | sed -e 's|serviceaccount\/|crossplane-system:|g')
-# kubectl create clusterrolebinding provider-kubernetes-admin-binding --clusterrole cluster-admin --serviceaccount="${SA}"
 apiVersion: kubernetes.crossplane.io/v1alpha1
 kind: ProviderConfig
 metadata:
   name: kubernetes-provider
 spec:
   credentials:
     source: InjectedIdentity
+  runtimeConfigRef:
+    apiVersion: pkg.crossplane.io/v1beta1
+    kind: DeploymentRuntimeConfig
+    name: provider-kubernetes
```

Comment:
I think this needs to be on the Provider, not on the ProviderConfig

Reply:
Small hickup during commit from flight ...